### PR TITLE
Feat zawolanski

### PR DESCRIPTION
**Pull request by zawolanski**
I'm added icons from figma, create Button(only with icon, regarding submit button we have to do it separately) component and i'm applied Poppins font.

### CODE REVIEW

File: src/theme/theme.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ const theme = {
     text_blue: '#0E84BF',
     profile_blue: '#003B59',
   },
+  bp: {
```
Comment (pawelsobkowski) [inline src/theme/theme.js]: Clearer name
```suggestion
  breakingPoints: {
```

Comment (zawolanski) [reply src/theme/theme.js]: Ok, i will change it

File: src/theme/GlobalStyles.js [hunk 2]
```diff
@@ -1,10 +1,23 @@
 import { createGlobalStyle } from 'styled-components';
 
 const GlobalStyles = createGlobalStyle`
+    :root {
+        font-family: 'Poppins', sans-serif;
+        font-size: 62.5%;
```
Comment (pawelsobkowski) [inline src/theme/GlobalStyles.js:6]: I think that it's better to use a `em` or `rem` unit.

File: src/views/Root/index.js [hunk 3]
```diff
@@ -15,17 +15,17 @@ const Root = () => (
...
           <Home />
         </Header>
       </Route>
-      <Route exact path="/dashboard">
+      <Route path="/dashboard">
         <Header>
           <Dashboard />
         </Header>
       </Route>
-      <Route exact path="/details/:placeId">
```
Comment (pawelsobkowski) [inline src/views/Root/index.js:28]: Why not `exact` attribute?

File: src/theme/theme.js (hunk 1 above)
Comment (pawelsobkowski) [reply src/theme/theme.js]: Thanks

File: src/theme/GlobalStyles.js (hunk 2 above)
Comment (zawolanski) [reply src/theme/GlobalStyles.js:6]: Yes we will use it. It's only reset on html element that let us easier calculate `rem`. Setting it up we get `1rem` = `10px`.

File: src/views/Root/index.js (hunk 3 above)
Comment (zawolanski) [reply src/views/Root/index.js:28]: If we want to get something like on image belowe we have to let router reach e.g `/home` and `/home/signin` at the same time
image

Comment (pawelsobkowski) [reply src/views/Root/index.js:28]: I think that sign in view will be display on foreground, without separate route for this

Comment (zawolanski) [reply src/views/Root/index.js:28]: Ok, but we still have `/search` route that is displaying with `/home` and `/dashboard` route.

**Pull request merged**